shared/src/test/scala/scorex/ScalaJsSpec.scala

```diff
@@ -98,6 +108,7 @@ class ScalaJsSpec extends AnyPropSpec with Matchers {
     t.toString() shouldBe "java.lang.String"
   }
 
+// Not supported
```

Comment:
remove commented out code or make the test ignored, with TODO, if needed

Reply:
It is not really something to do. I explained in ScalaDoc for ScalaJsSpec what this `// Not supported` comments mean, so it is not something to be cleaned up as this code is useful for future maintenance.
If `// Not supported` is not clear enough, please suggest an alternative wording.